crmd/lrm.c

```diff
@@ -2077,6 +2077,19 @@ do_lrm_rsc_op(lrm_state_t * lrm_state, lrmd_rsc_info_t * rsc, const char *operat
         }
     }
 
+    if (!is_remote_lrmd_ra(NULL, NULL, rsc->id)) {
```

Comment:
Is there a reason to exclude the remote ra here? I don't expect needing versioned attributes for it, but I think it would work fine.